main.js

```diff
@@ -0,0 +1,41 @@
+let user = 'John Doe';
+const student = 'Yauheniya';
+console.log(student);
+user = student;
+// in user now we have value 'Yauheniya'
+console.log(user);
+let test = 1;
+test++;
+test += '1';
+console.log(test);
+//in test now we have value: 2 and '1'(like text) but shows 21 (cause it mains to one type of data - string)
+test--;
+// in test now we have value: 20 (cause it mains to one type of data - number)
+console.log(test);
+test = true;
+// in test now we have: true
+console.log(test);
+let result = 1;
+const mv = [2, 3, 5, 8];
+for (let i = 0; i < mv.length; i++) {
+  result = result * mv[i];
+}
+console.log(result);
+let result1 = '';
+const y = [2, 5, 8, 15, 0, 6, 20, 3];
+let t = 0;
```

Comment:
You may put it into for credentials instead of missing the first there.
`for (let i = 0; i < y.length; i++)` looks better.